manifest.yml

```diff
@@ -0,0 +1,19 @@
+---
+# This manifest deploys a Python Flask application with a Cloudant database
+applications:
+- name: nyu-wishlist-service-fall2103
+  path: .
+  instances: 2
+  memory: 512M
+  routes:
+  - route: nyu-wishlist-service-fall2103.us-south.cf.appdomain.cloud
+  disk_quota: 1024M
+  buildpacks: 
+  - python_buildpack
+  timeout: 180
+  services:
+  - ElephantSQL
+  - ElephantSQL_test
+  env:
+    FLASK_APP : service:app
+    FLASK_DEBUG : false
```

Comment:
Could you make sure to add new line at the end of all files please? linter barks at this. There is a VS code setting to do it automatically for you when you save a file:
<img width="1001" alt="Screen Shot 2021-11-11 at 11 25 03 AM" src="https://user-images.githubusercontent.com/33898347/141332976-fc4e8450-ea17-4db0-83d8-7dbc705b1c4a.png">